### ISSUE TITLE
Create labeled video with .csv file

### ISSUE THREAD

**Issue by jennymsmith94** [DeepLabCut/DeepLabCut#148]
Windows 10 with an Anaconda Env & DeeplabCut2.?

TL;DR- Is it possible to use `create_labeled_video` with the `.csv` file?

Not a problem, but a request from a python and ML newbie.

I successfully trained a network to predict position of rats on a y-maze with pretty grainy video quality. It works really well for like 80% of the video but there's a lot of jumping around. It's usually just a couple of frames and then back to the correct body part. I am very comfortable in MatLab since most of our data analysis is done there, so I did a little smoothing on the position data from the `.csv` in MatLab. When I tried to use the `create_labeled_video` function, I realized that uses the `.h5` data to make the video and there doesn't appear to be an option to use the `.csv`, which would be really useful for me.

I could probably figure out a way to use the `.csv` (or just get more familiar with python and `.h5`) but was wondering if anyone else would find a `.csv` option useful or had an easy workaround/advice.

Thanks!

**Comment by MMathisLab** [comment]
Seems if it’s jumping you’d benefit from extracting outliers and refining your network... (see user guide).

You could covert the csv to h5; many solutions online with a quick google search. Also questions (vs issues) are likely to get more feedback on the forum: https://forum.image.sc/tags/deeplabcut

**Comment by MMathisLab** [comment]
You could covert the csv to h5; many solutions online with a quick google search. Also questions (vs issues) are likely to get more feedback on the forum: www.forum.image.sc

**Comment by jennymsmith94** [comment]
Ahh I see.. thank you!
